src/util/postgres_client.js

```diff
@@ -91,7 +92,7 @@ function encode_json(schema, val) {
     if (!val || !schema) {
         return val;
     }
-    // eslint-disable-next-line no-use-before-define
```

Comment:
removing this line is causing eslint errors, at least in my env. I'm not sure why it is not failing eslint in the tests

Reply:
Hi Danny,
I run `eslint` on the file and I don't get an error.
Just to be sure `eslint` works I changed the header (2016 -> zzz) and saved and got an error.
Change it back - and nothing...
![Screenshot 2023-01-19 at 8 23 18](https://user-images.githubusercontent.com/57721533/213370528-dd52d277-c81c-4208-9e39-420da60960ab.png)

Reply:
1. `eslint` is running in the required checks and it passed (runs `npm run lint`), as mentioned above and in the CI/CD meeting.
2. After fetching the most updated code, including #7158  this was not an error on Danny's machine as well.